Source/Android/app/src/main/res/values/arrays.xml

```diff
@@ -326,12 +326,14 @@
         <item>GFX Settings</item>
         <item>GameCube Controller Settings</item>
         <item>Clear Game Settings</item>
+        <item>Set as default ISO</item>
```

Comment:
The capitalization doesn't match the other options.

Reply:
Think I should capitalize "As" too?

Reply:
No.